File: code3.py
n=int(input())
for i in range(n):
    s=input()
    s1=input()
    count=0
    for i in range(len(s)):
        for j in range(len(s1)):
            if(s[i]==s1[j]):
                count=count+1
                break
        if(count==len(s)):
            print("YES")
            break
    if(count<len(s)):
        print("NO")

File: code3.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		sc.nextLine();

		for (int k = 0; k < n; k++) {
			String s = sc.nextLine();
			String s1 = sc.nextLine();
			int count = 0;
			for (int i = 0; i < s.length(); i++) {
				for (int j = 0; j < s1.length(); j++) {
					if (s.charAt(i) == s1.charAt(j)) {
						count++;
						break;
					}
				}
				if (count == s.length()) {
					System.out.println("YES");
					break;
				}
			}
			if (count < s.length()) {
				System.out.println("NO");
			}

		}

	}
}
